jf_agent/jf_jira/__init__.py

```diff
@@ -61,6 +62,18 @@ def print_all_jira_fields(config, jira_connection):
         print(f"{f['key']:30}\t{f['name']}")
 
 
+@diagnostics.capture_timing()
+@agent_logging.log_entry_exit(logger)
+def print_missing_repos_found_by_jira(issues_to_scan, config, jira_connection, git_connection):
+    missing_repos = download_missing_repos_found_by_jira(
+        issues_to_scan, config, jira_connection, git_connection
+    )
+    print(f'\nJira found {len(missing_repos)} repos potentially missing from Jellyfish')
```

Comment:
Possible alternate suggestions:

 - Scanning the "Development" field on the Jira issues revealed XX Git repos apparently missing from Jellyfish

 - Jira issues refer to XX Git repos that are missing from Jellyfish